### DISCUSSION TITLE
How to configure axway openlogging on an existing ELK stack?

### DISCUSSION THREAD

**Discussion by vishnuindla** [Axway-API-Management-Plus/apigateway-openlogging-elk#185] [Q&A]
Hello team,

I have installed ELK stack using a docker image and now we are looking at trying to use that to configure openlogging for Axway. This image already has Elastic search, Kibana and Logstash configured on it. Just wondering if there is an easy way to install apibuilder4elastic separately and point to an existing ELK stack? Please let me know.

**Comment by cwiechmann** [comment]
The solution supports an external Elasticsearch-Cluster + Kibana, but Logstash and it's configuration is a core component of this solution. Therefore, please deploy Logstash as part of this solution, as it's described here:
https://github.com/Axway-API-Management-Plus/apigateway-openlogging-elk#logstash--api-builder--memcached

**Reply by vishnuindla** [reply]
Hi @cwiechmann ,

Thanks for your reply. We will give that a try. I also wanted to check one more thing. We are planning to build our own image instead of using the docker image from the community. Is there a Dockerfile that we can use to create our own linux image with apibuilder/logstash ?

Thanks
Vishnu

**Reply by cwiechmann** [reply]
For support reasons, it's not planned, that customers build their own Docker-Images. The solution has been tested, in terms of performance and function based on the baked Docker-Images, that are part of the release packages. Customer-Specific Docker-Images would introduce a number of unknowns, that would require intensive testing, esp. for performance, to become supportable.

For the entire Elasticstack, the solution is using the standard Elastic-Images, incl. Logstash.
